haskell-language-server.cabal

```diff
@@ -1792,225 +1819,236 @@ common notes
-  exposed-modules:
-    Ide.Plugin.Notes
-  hs-source-dirs:     plugins/hls-notes-plugin/src
-  build-depends:
-    , base >=4.12 && <5
-    , array
-    , ghcide == 2.8.0.0
-    , hls-graph == 2.8.0.0
-    , hls-plugin-api == 2.8.0.0
-    , lens
-    , lsp >=2.5
-    , mtl >= 2.2
-    , regex-tdfa >= 1.3.1
-    , text
-    , text-rope
-    , unordered-containers
+  exposed-modules: Ide.Plugin.Notes
+  hs-source-dirs: plugins/hls-notes-plugin/src
+  build-depends:
+    array,
+    base >=4.12 && <5,
+    ghcide ==2.8.0.0,
+    hls-graph ==2.8.0.0,
+    hls-plugin-api ==2.8.0.0,
+    lens,
+    lsp >=2.5,
+    mtl >=2.2,
+    regex-tdfa >=1.3.1,
+    text,
+    text-rope,
+    unordered-containers,
+
   default-extensions:
-      DataKinds
-    , DeriveAnyClass
-    , DerivingStrategies
-    , OverloadedStrings
-    , LambdaCase
-    , TypeFamilies
+    DataKinds
+    DeriveAnyClass
+    DerivingStrategies
+    LambdaCase
+    OverloadedStrings
+    TypeFamilies
 
 test-suite hls-notes-plugin-tests
-  import:           defaults, pedantic, test-defaults, warnings
+  import: defaults, pedantic, test-defaults, warnings
+
   if !flag(notes)
     buildable: False
-  type:             exitcode-stdio-1.0
-  hs-source-dirs:   plugins/hls-notes-plugin/test
-  main-is:          NotesTest.hs
-  build-depends:
-    , base
-    , directory
-    , filepath
-    , haskell-language-server:hls-notes-plugin
-    , hls-test-utils == 2.8.0.0
+  type: exitcode-stdio-1.0
+  hs-source-dirs: plugins/hls-notes-plugin/test
+  main-is: NotesTest.hs
+  build-depends:
+    base,
+    directory,
+    filepath,
+    haskell-language-server:hls-notes-plugin,
+    hls-test-utils ==2.8.0.0,
+
   default-extensions: OverloadedStrings
 
 ----------------------------
 ----------------------------
 -- HLS
 ----------------------------
 ----------------------------
-
 library
-  import:           defaults
-                  , warnings
-                  , pedantic
-                  -- plugins
-                  , cabal
-                  , callHierarchy
-                  , cabalfmt
-                  , cabalgild
-                  , changeTypeSignature
-                  , class
-                  , eval
-                  , importLens
-                  , rename
-                  , retrie
-                  , hlint
-                  , stan
-                  , moduleName
-                  , pragmas
-                  , splice
-                  , alternateNumberFormat
-                  , qualifyImportedNames
-                  , codeRange
-                  , gadt
-                  , explicitFixity
-                  , explicitFields
-                  , floskell
-                  , fourmolu
-                  , ormolu
-                  , stylishHaskell
-                  , refactor
-                  , overloadedRecordDot
-                  , semanticTokens
-                  , notes
+  import:
+    defaults
+    , warnings
+    , pedantic
+    -- plugins
```

Comment:
@tfausak here! This comment is (surprisingly?) preserved in situ.

Reply:
Starting with [version 1.4.0.0](https://github.com/tfausak/cabal-gild/releases/tag/1.4.0.0), Gild will always float comments up to the top of fields.